src/Organisms/RegistrationForm/RegistrationForm.js

```diff
@@ -0,0 +1,25 @@
+import { useState } from 'react';
+
+import RegistrationFormSignup from './RegistrationFormSignup';
+import RegistrationFormSuccess from './RegistrationFormSuccess';
+import ColumnWrapper from '../../Templates/ColumnWrapper/ColumnWrapper';
+
+const RegistrationForm = () => {
+  const [isSubmitted, setIsSubmitted] = useState(false);
+
+  function submitForm() {
+    setIsSubmitted(true);
```

Comment:
It shuold probably check in the future if it was really sent, and only then mark as true